drafts/use-cases/5.creating-new-event.md

```diff
@@ -0,0 +1,88 @@
+## Creating a new event
+
+### Story
```

Comment:
As discussed, please keep both the abstract story plus the concrete example. Thanks